server/services/datacommons.py

```diff
@@ -328,6 +328,23 @@ def get_property_labels(dcids):
     return results
 
 
+def property_values(entities, prop, dir):
```

Comment:
It is a good idea to document what this function returns. Ideally, one shouldn't need to parse the function implementation to understand what's expected :)

Reply:
Got it! Will add

Reply:
I actually just merged in master and this is no longer a diff.  I'll address this in the backend pr since it fits better there.